lib/extensions/extensions.dart

```diff
@@ -1,10 +1,11 @@
-export 'firebase_user_extensions.dart';
```

Comment:
this is just the file that allows easy import of all extensions

Reply:
thanks 🌮